src/guildhouse.h

```diff
@@ -0,0 +1,9 @@
+
+// Offsets from creatures_objects.sql
+constexpr uint32 GetCreatureEntry(uint32 offset) {
+    return 500030 + offset;
```

Comment:
Okay, but what if I set  @C_TEMPLATE = 40050, then everything would break.

Reply:
Same as before.
But now you can easily change it in this one place.
If you change it in SQL script, you need to change it here too.

But atleast there are only 4 places to change, instead of two dozen as before

Reply:
well then put it into the config. Hardcoded stuff are bad.